modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+
+==== If URL paths contained either a colon or an exclamation point (or both) PowerPaste’s link detection logic failed
+
+If a colon or an exclamation point — `+:+` or `+!+` was present in the path portion of a URL, PowerPaste’s link detection logic treated the characters as, in effect, end-of-path boundaries. As a consequence, while the string was pasted into {productname}, the expected link was incomplete and incorrect.
+
+For example, pasting the string
+
+`https://example.com/!exclamation-point`
+
+resulted in the following, incomplete, link and trailing text
+
+https://example.com/[]!exclamation-point
+
+Equivalent results occured when URL paths contained the colon character (for example, `https://example.com/:colon`).
+
+With this update, the link detection logic has been improved to capture and present such URLs correctly.
+
+Now, pasting the string 
+
+`https://example.com/!exclamation-point`
+
+results in the following, complete, link and no trailing text
+
+https://example.com/!exclamation-point[]
+
+Also, this improved link detection logic works when manually typing in a URL. If the above example were typed in to a {productname} document, the automatic link-creation would wrap the entire string in an anchor tag after a boundary character (such as a space) was typed.
+
+NOTE: exclamation points and colons are still boundary characters for URL detection and delmiting. If the string `https://example.com/colon:` is pasted or typed into a {productname} document, automatic link detection will produce a link that does not include the trailing colon character.
+
+
+==== Pasted URL-like strings without a defining protocol were detected as URLs but not provided with a protocol
+
+When fully qualified domain names (FQDN), such as `www.example.com`, were pasted in to a {productname} document, they were detected as URLs, but no protocol was added to the consequent underlying `href` attribute.
+
+As a consequence, the text would display as a clickable link but not work as such.
+
+With this release such FQDNs are, when pasted in to place, supplied with {productname}’s default protocol — `https` — in the underlying `href` attribute. And these pasted strings now display as a clickable link and works as such, as expected.
+
+For information on the **PowerPaste** plugin, see: xref:powerpaste.adoc[PowerPaste plugin].
+
+
+=== Real-Time Collaboration 2.1.0
+
+The {productname} 6.1 release includes an accompanying release of the **Real-Time Collaboration** premium plugin.
+
+**Real-Time Collaboration** 2.1.0 adds one new features and fixes two bugs:
+
+==== Real-Time Collaboration can now disable inline boundaries
+
+xref:content-behavior-options.adoc#inline_boundaries[*Inline boundaries*] are highlights applied to certain inline elements. The exact elements highlighted can be configured with `inline_boundaries_selector` but the elements highlighted by default are `a` (anchor tags), `code`, and `.mce-annotation`.
```

Comment:
Should `inline_boundaries_selector` be a link to the option as well here?

Also I don't think we should be mentioning the defaults here, as that could get outdated easily and then lead to confusion.

Reply:
Added a link to `inline_boundaries_selector` documentation and removed the reference to extant defaults.